File: java_backup/java_course/fact.java
class fact
{
    int i,f,n1;
    fact(int n)
    {
        f=1;
        n1=n;
    }
    void fact()
    {
        for(i=1;i<=n1;i++)
        {
            f=f*i;
        }
        System.out.println(f);
    }
}

File: python/python_backup/Jimut/fact.py
from tqdm import tqdm

def fact(n):                                                                                                                                                                    #function to calculate factorial
        f0=1
        a0=0
        for i0 in tqdm(range(n)):
                a0=i0+1
                f0=f0*a0
        
        print("The factorial of ",a0)
        print(f0)

x=input("Enter the number which you want to make the factorial of ? :")

fact(x)

